src/brushes/SprayBrush.ts

```diff
@@ -143,6 +144,7 @@ export class SprayBrush extends BaseBrush {
         objectCaching: true,
         subTargetCheck: false,
         interactive: false,
+        layoutManager: new LayoutManager(new FixedLayout()),
```

Comment:
can you explain this change? doesn't seem related to setCoords

Reply:
Not very, true
No need for the spray brush to have a fit content layout